SingularityBase/src/main/java/com/hubspot/singularity/SingularityDeploy.java

```diff
@@ -56,6 +56,7 @@
   private final Optional<String> serviceBasePath;
   private final Optional<List<String>> loadBalancerGroups;
   private final Optional<Map<String, Object>> loadBalancerOptions;
+  private final Optional<String> previousName;
```

Comment:
`previousName` is too ambiguous for my taste -- i'd follow the Baragon field naming and call it something like `loadBalancerReplaceServiceId`